src/erlcloud_states.erl

```diff
@@ -539,7 +539,7 @@ start_execution(StateMachineArn, Options) ->
     start_execution(StateMachineArn, Options, default_config()).
 
 -spec start_execution(StateMachineArn   :: binary(),
-                      Options           :: list(),
+                      Options           :: list() | map(),
```

Comment:
It looks like it's just "map" only, no?

Reply:
Yup, you're right. I saw the function was exported and though I didn't want to break the external interface, but it does guard against maps, though.

Reply:
Fixed in 067807c.